src/main/java/me/EvsDev/EnderDragonTweaks/ConfigManager.java

```diff
@@ -9,11 +10,13 @@
 import org.bukkit.util.Vector;
 
 import net.md_5.bungee.api.ChatColor;
+import org.bukkit.Location;
 
 public class ConfigManager {
 
 	public static final String entry_enabled = "enabled";
 	public static final String entry_delay = "delay";
+        public static final String entry_playerDistanceFromOrigin = "max-player-distance-from-end-center";
```

Comment:
Funky indent - see general review.
```suggestion
    public static final String entry_playerDistanceFromOrigin = "max-player-distance-from-end-centre";
```